src/Sylius/Bundle/AttributeBundle/AttributeType/CheckboxAttributeType.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\AttributeType;
+
+use Sylius\Component\Attribute\AttributeType\AttributeTypeInterface;
+use Sylius\Component\Attribute\AttributeType\DefaultAttributeTypes;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class CheckboxAttributeType implements AttributeTypeInterface
+{
+    /**
+     * @return string
+     */
+    public function getStorageType()
+    {
+        return DefaultAttributeTypes::STORAGE_BOOLEAN;
+    }
+
+    /**
+     * @return string
+     */
+    public function getType()
+    {
+        return DefaultAttributeTypes::CHECKBOX;
```

Comment:
I think these constants should be on *AttributeType classes
Storage type constants should live on AttributeValueInterface, because this is where we have storages defined. Wdyt?